packages/esm-commons-lib/src/components/encounter-list/helpers.ts

```diff
@@ -1,28 +1,33 @@
-import { OHRIFormSchema, SessionMode } from '@openmrs/openmrs-form-engine-lib';
-import { launchForm } from '../../utils/ohri-forms-commons';
-import { capitalize } from 'lodash-es';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
 
 type LaunchAction = 'add' | 'view' | 'edit';
 
 export function launchEncounterForm(
-  form: OHRIFormSchema,
+  form: any,
```

Comment:
The `OHRIFormSchema` comes from the form engine. Are there any missing values that necessitated this change?

Reply:
yes, but i can look into finding a workaround